models.yml

```diff
@@ -14,6 +14,409 @@ basel:
+        - 'DE-BE'
+        - 'DE-BB'
+        - 'DE-HB'
+        - 'DE-HH'
+        - 'DE-HE'
+        - 'DE-MV'
+        - 'DE-NI'
+        - 'DE-NW'
+        - 'DE-RP'
+        - 'DE-SL'
+        - 'DE-SN'
+        - 'DE-ST'
+        - 'DE-SH'
+        - 'DE-TH'
+    - GH
+    - GR
+    - GL
+    - GD
+    - GT
+    - GN
+    - GW
+    - GY
+    - HT
+    - HN
+    - HK
+    - HU
+    - IS
+    - IN:
+        - 'IN-AN'
+        - 'IN-AP'
+        - 'IN-AR'
+        - 'IN-AS'
+        - 'IN-BR'
+        - 'IN-CH'
+        - 'IN-CT'
+        - 'IN-DN'
+        - 'IN-DD'
+        - 'IN-DL'
+        - 'IN-GA'
+        - 'IN-GJ'
+        - 'IN-HR'
+        - 'IN-HP'
+        - 'IN-JK'
+        - 'IN-JH'
+        - 'IN-KA'
+        - 'IN-KL'
+        - 'IN-LD'
+        - 'IN-MP'
+        - 'IN-MH'
+        - 'IN-MN'
+        - 'IN-ML'
+        - 'IN-MZ'
+        - 'IN-NL'
+        - 'IN-OR'
+        - 'IN-PY'
+        - 'IN-PB'
+        - 'IN-RJ'
+        - 'IN-SK'
+        - 'IN-TN'
+        - 'IN-TH'
+        - 'IN-TR'
+        - 'IN-UP'
+        - 'IN-UT'
+        - 'IN-WB'
+    - ID
+    - IR
+    - IQ
+    - IE
+    - IL
+    - IT:
+        - 'IT-65'
+        - 'IT-77'
+        - 'IT-78'
+        - 'IT-72'
+        - 'IT-45'
+        - 'IT-36'
+        - 'IT-62'
+        - 'IT-42'
+        - 'IT-25'
+        - 'IT-57'
+        - 'IT-67'
+        - 'IT-BZ'
+        - 'IT-TN'
+        - 'IT-21'
+        - 'IT-75'
+        - 'IT-88'
+        - 'IT-82'
+        - 'IT-52'
+        - 'IT-32'
+        - 'IT-55'
+        - 'IT-23'
+        - 'IT-34'
+    - JM
+    - JP:
+        - 'JP-47'
+    - JO
+    - KZ
+    - KE
+    - KI
+    - KP
+    - KR
+    - KW
+    - KG
+    - LA
+    - LV
+    - LB
+    - LR
+    - LY
+    - LI
+    - LT
+    - LU
+    - MG
+    - MW
+    - MY
+    - MV
+    - ML
+    - MT
+    - MH
+    - MU
+    - MX
+    - FM
+    - MD
+    - MC
+    - MN
+    - ME
+    - MA
+    - MZ
+    - MM
+    - NA
+    - NR
+    - NP
+    - NL
+    - NZ
+    - NI
+    - MK
+    - NO
+    - OM
+    - PK
+    - PW
+    - PA
+    - PY
+    - PE
+    - PH:
+        - 'PH-14'
+        - 'PH-15'
+        - 'PH-00'
+        - 'PH-01'
+        - 'PH-10'
+        - 'PH-11'
+        - 'PH-12'
+        - 'PH-13'
+        - 'PH-02'
+        - 'PH-03'
+        - 'PH-40'
+        - 'PH-41'
+        - 'PH-05'
+        - 'PH-06'
+        - 'PH-07'
+        - 'PH-08'
+        - 'PH-09'
+    - PL
+    - PT
+    - QA
+    - RO
+    - RU
+    - KN
+    - LC
+    - VC
+    - SM
+    - ST
+    - SA
+    - SN
+    - RS
+    - SC
+    - SG
+    - SK
+    - SI
+    - SB
+    - SO
+    - ZA
+    - ES:
+        - 'ES-AN'
+        - 'ES-AR'
+        - 'ES-AS'
+        - 'ES-IB'
+        - 'ES-VC'
+        - 'ES-CN'
+        - 'ES-CB'
+        - 'ES-CL'
+        - 'ES-CM'
+        - 'ES-CT'
+        - 'ES-CE'
+        - 'ES-EX'
+        - 'ES-GA'
+        - 'ES-RI'
+        - 'ES-MD'
+        - 'ES-ML'
+        - 'ES-MC'
+        - 'ES-NC'
+        - 'ES-PV'
+    - LK
```

Comment:
Boring suggestion: sort the codes in alphabetical order (I assume it's currently something like country name order). This will keep it tidier as we add new regions or look for existing regions.